src/main/resources/db/changelog/v.0.5/db.changelog-time-for-call-user.yaml

```
@@ -0,0 +1,33 @@
+databaseChangeLog:
+  - changeSet:
+      id: 25042021-add-table-time-for-call-user
+      author: nikita_bogurov
+      changes:
+        - createTable:
+            tableName: time_for_call_user
```

```suggestion
            tableName: user_time_slot
```
@SergeyKoval @viktarkolbik your proposal?

---

Done